.goreleaser-linux-amd64.yml

```
@@ -34,6 +34,33 @@ builds:
     goarch:
       - amd64
 
+nfpms:
+  - package_name: confluent-cli
+    file_name_template: "{{ .ConventionalFileName }}"
+    builds:
+      - confluent-linux-amd64-disableupdates
+    vendor: Confluent Inc.
```

I'm pretty sure there's usually a comma here
```suggestion
    vendor: Confluent, Inc.
```